Makefile

```diff
@@ -14,6 +14,10 @@ export IMAGE_TAG
 
 IMG := $(IMAGE_REPO):$(IMAGE_TAG)
 
+# Setting SHELL to bash allows bash commands to be executed by recipes.
```

Comment:
Do these lines need to be even higher, e.g. do `origin` calls use the shell?

Reply:
Probably doesn't hurt to stick it all the way at the top anyways

Reply:
`$(origin)` returns the origin of a variable; it doesn't use the shell, but I can put it right at the top.